cellfinder_napari/utils.py

```diff
@@ -135,6 +136,44 @@ def add_combobox(
     return combobox, combobox_label
 
 
+def add_spinbox(
+    layout: QLayout,
+    label: str,
+    connected_function: Callable,
+    row: int,
+    column: int = 0,
+    label_stack: bool = False,
+    width: int = 150,
+) -> QSpinBox:
+    """
+    Defines the properties for the SpinBox Widget.
+    """
+    if label_stack:
+        spinbox_row = row + 1
+        spinbox_column = column
+    else:
+        spinbox_row = row
+        spinbox_column = column + 1
+    spinbox = QSpinBox()
+    spinbox.setSingleStep(2)
+    spinbox.setMaximumWidth = width
+    spinbox.setKeyboardTracking(False)
+
+    spinbox_label = QLabel(label)
+    spinbox.setMaximumWidth = width
+    if label == "Cube Depth":
+        spinbox.setValue(20)
+        spinbox.setRange(1, 20)
+    else:
+        spinbox.setValue(50)
+        spinbox.setRange(1, 50)
```

Comment:
Same comment as above. Theoretically, someone could have v. high resolution data (or huge objects to detect) and might need bigger cuboids.